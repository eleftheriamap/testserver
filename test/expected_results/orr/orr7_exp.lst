
test/exp/test/gen/orr/orr7_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	51648c21 	sub	w1, w1, #0x923, lsl #12
   4:	9166bfbd 	add	x29, x29, #0x9af, lsl #12
   8:	2a9d0022 	orr	w2, w1, w29, asr #0
   c:	8a000000 	and	x0, x0, x0
