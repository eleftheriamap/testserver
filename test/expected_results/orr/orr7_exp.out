Registers:
X00    = 0000000000000000
X01    = 00000000ff6dd000
X02    = 00000000fffff000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 00000000009af000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x51648c21
0x00000004 : 0x9166bfbd
0x00000008 : 0x2a9d0022
0x0000000c : 0x8a000000
